indigo_app/static/javascript/indigo/views/document_comparisons.js

```diff
@@ -0,0 +1,66 @@
+(function(exports) {
+  "use strict";
+
+  if (!exports.Indigo) exports.Indigo = {};
+  Indigo = exports.Indigo;
+
+  /**
+   * Compare different expressions (in the same language) of a work.
+   */
+  Indigo.DocumentComparisonView = Backbone.View.extend({
+    el: '.document-comparison-view',
+    comparisonHeaderTemplate: '#comparison-header-template',
+    events: {
+      'click .dismiss': 'dismiss'
+    },
+
+    initialize: function(options) {
+      this.comparisonHeaderTemplate = Handlebars.compile($(this.comparisonHeaderTemplate).html());
+      this.document = options.document;
+      this.documentContent = options.documentContent;
+      this.expressions = new Indigo.Library(Indigo.Preloads.expressions);
+
+      $('.menu .comparison').on('click', _.bind(this.show, this));
+    },
+
+    comparison: function(comparison_doc_id) {
```

Comment:
Style: in Javascript, the norm is to use camelCasedVariableNames, whereas python uses underscore_variable_names.